unison-src/transcripts/diff-namespace.output.md

```diff
@@ -189,26 +184,26 @@ unique type Y a b = Y a b
   Added definitions:
   
     8.  type Y a b
-    9.  Y.Y : a -> b -> Y a b
+    9.  Y.Y : a -> b -> #md85ksgqel a b
```

Comment:
broken?

Reply:
Turns out this is an existing bug in update.old that is simply manifesting here; we should port this to use `update` instead.